README.md

```diff
@@ -39,7 +39,7 @@ You'll notice that we are using container and presentational components to separ
 - The purpose of this mini-project is to expose you to the world of asyncronous Redux.
```

Comment:
Compiled with warnings.

./src/views/CharacterListView.js
  Line 8:   Useless constructor                                                    no-useless-constructor
  Line 21:  Expected an assignment or function call and instead saw an expression  no-unused-expressions